code/modules/mob/living/simple_animal/hostile/terror_spiders/empress.dm

```diff
@@ -0,0 +1,241 @@
+// -------------: AI: none - this is strictly adminspawn-only and intended for RP events, coder testing, and teaching people 'how to queen'
+// -------------: SPECIAL: Lay Eggs ability that allows laying queen-level eggs. Wide-area EMP and light-breaking abilities.
+// -------------: TO FIGHT IT: run away screaming?
+// -------------: CONCEPT: http://tvtropes.org/pmwiki/pmwiki.php/Main/EldritchAbomination
+// -------------: CONCEPT: http://tvtropes.org/pmwiki/pmwiki.php/Main/BiggerBad
+// -------------: CONCEPT: http://tvtropes.org/pmwiki/pmwiki.php/Main/AuthorityEqualsAsskicking
+// -------------: SPRITES FROM: FoS, http://nanotrasen.se/phpBB3/memberlist.php?mode=viewprofile&u=386
+
+/mob/living/simple_animal/hostile/poison/terror_spider/empress
+	name = "Empress of Terror"
+	desc = "The unholy offspring of spiders, nightmares, and lovecraft fiction."
+	spider_role_summary = "Adminbus spider"
+	altnames = list ("terror empress spider")
+	egg_name = "empress spider eggs"
+
+	icon_state = "terror_queen"
+	icon_living = "terror_queen"
+	icon_dead = "terror_queen_dead"
+
+	maxHealth = 1000
+	health = 1000
+
+	melee_damage_lower = 10
+	melee_damage_upper = 40
+
+	move_to_delay = 5
+	ventcrawler = 1 // Adminbus.
+
+	idle_ventcrawl_chance = 0
+	ai_playercontrol_allowtype = 0
+	ai_type = 1 // defend self only!
+
+	ranged = 1
+	retreat_distance = 5
+	minimum_distance = 5
+	projectilesound = 'sound/weapons/pierce.ogg'
+	projectiletype = /obj/item/projectile/terrorempressspit
+	force_threshold = 18 // same as queen, but a lot more health
+
+	spider_tier = 5
+	spider_opens_doors = 2
+
+	var/shown_guide = 0 // has the empress player been warned of the chaos that can result from the use of their powers?
+
+	gold_core_spawnable = CHEM_MOB_SPAWN_INVALID
+
+
+/mob/living/simple_animal/hostile/poison/terror_spider/empress/New()
+	..()
+	spawn(600)
+		if (!ckey)
+			// idea being that if someone spawns her, and she isn't player controlled within 60 seconds, it was probably a mistake and she should be despawned.
+			visible_message("[src] steps into a bluespace portal, and is gone!")
+			qdel(src)
+
+/mob/living/simple_animal/hostile/poison/terror_spider/empress/Life()
+	if (stat != DEAD)
+		if (ckey && !shown_guide)
+			shown_guide = 1
+			ShowGuide()
+	..()
+
+
+/mob/living/simple_animal/hostile/poison/terror_spider/empress/verb/LayEmpressEggs()
+	set name = "Lay Empress Eggs"
+	set category = "Spider"
+	set desc = "Lay spider eggs. As empress, you can lay queen-level eggs to create a new brood."
+	var/eggtype = input("What kind of eggs?") as null|anything in list("QUEEN", "MOTHER", "PRINCE", "red - assault","gray - ambush", "green - nurse", "black - poison","purple - guard")
+	var/numlings = input("How many in the batch?") as null|anything in list(1,2,3,4,5,10,15,20,30,40,50)
+	if (eggtype == null || numlings == null)
+		to_chat(src, "Cancelled.")
+		return
+	// T1
+	if (eggtype == "red - assault")
```

Comment:
They need to be in the INPUT, I don't think you can use DEFINEs for the input()?